redash/extensions.py

```diff
@@ -10,5 +10,8 @@ def init_extensions(app):
 
     for entry_point in iter_entry_points('redash.extensions'):
         app.logger.info('Loading Redash extension %s.', entry_point.name)
-        extension = entry_point.load()
-        app.redash_extensions[entry_point.name] = extension(app)
+        try:
+            extension = entry_point.load()
+            app.redash_extensions[entry_point.name] = extension(app)
```

Comment:
Nice, this is a lot smaller than I thought. I think it would make sense to still store something for extensions that raised an ImportError on `ep.load()`, but only if their `bundle` directory exists. That way this covers all four cases:

- backend extensions (stores the return value of the callable)
- hybrid extensions (same as backend extension)
- frontend extensions (stores a list of files in the bundle directory?)
- misconfigured extensions (doesn't show up in `app.redash_extensions`)

The reason to having all extensions in `app.redash_extensions` would be an extension UI in the future where admins can see which extensions are enabled. If we were to not store frontend extensions we'd lose that information and would have to redo the list again for display.